pages/atom.js

```diff
@@ -1,143 +1,74 @@
 // @flow
```

Comment:
@jdetle these examples

Reply:
Oops! Must have accidentally deleted that in a rebase. Good catch! Thanks.